config.py

```diff
@@ -1,7 +1,15 @@
 import logging
 import os
 
-COBOT_ROOT = os.environ.get('COBOT_ROOT', os.getcwd())
+BOT_ROOT_KEY = 'BOT_ROOT'
+
+# Deprecate COBOT_ROOT
+if 'COBOT_ROOT' in os.environ and BOT_ROOT_KEY not in os.environ:
+    logging.warning(
+        'Environment variable {} is replaced by {}.'
+        .format('COBOT_ROOT', BOT_ROOT_KEY))
+BOT_ROOT = os.environ.get(BOT_ROOT_KEY, os.getcwd())
```

Comment:
if `COBOT_ROOT` was set, you dont load it.

Reply:
Which line are you referring to? The 14th one? Or the warning?

Reply:
The warning. You just warn that it's deprecated, but don't use the old env-variable completely.

I would recommend to pick my message from an earlier comment so people know that the variable is still functional:
> "Environment variable {} is deprecated, use {} instead."

Reply:
:+1: